### PR TITLE
Rename a module with a different name to keep clippy happy

### PR DESCRIPTION
**Pull request by pimeys**
Sigh.

### CODE REVIEW

**Review by dpetrick — APPROVED** [review]

File: libs/sql-schema-describer/tests/describer_tests.rs [hunk 1]
```diff
@@ -435,7 +435,7 @@ fn defaults_must_work(api: TestApi) {
     let id = user_table.columns.first().unwrap();
 
     assert_eq!("id", &id.name);
-    assert_eq!(false, id.auto_increment);
+    assert!(!id.auto_increment);
```
Comment (janpio) [inline libs/sql-schema-describer/tests/describer_tests.rs:438]: Intentional unrelated change?

Comment (tomhoule) [reply libs/sql-schema-describer/tests/describer_tests.rs:438]: It behaves the same, but it's shorter, more readable and has a slightly better error message if it fails.
